src/ProgressBar.xml

```diff
@@ -6,29 +6,39 @@
     </icon>
     <properties>
-        <property key="progressAttribute" type="attribute">
+        <property key="progressAttribute" type="attribute" required="false">
             <caption>Progress attribute</caption>
-            <category>Data source</category>
-            <description>Holds the progress value</description>
+            <category>Values</category>
+            <description>When no attribute is provided the 'Static value' will be used to calculate the progress</description>
             <attributeTypes>
                 <attributeType name="Integer" />
                 <attributeType name="Long" />
                 <attributeType name="Decimal" />
             </attributeTypes>
         </property>
+        <property key="staticValue" type="integer" defaultValue="50">
+            <caption>Value static</caption>
+            <category>Values</category>
+            <description/>
+        </property>
         <property key="maximumValueAttribute" type="attribute" required="false">
             <caption>Maximum value attribute</caption>
-            <category>Data source</category>
-            <description>Highest progress value. If not set, maximum value defaults to 100</description>
+            <category>Values</category>
+            <description>Holds the highest progress value which must be greater than 0. If not set, the maximum default value is set to 'Maximum value static'</description>
             <attributeTypes>
                 <attributeType name="Integer" />
                 <attributeType name="Long" />
                 <attributeType name="Decimal" />
             </attributeTypes>
         </property>
+        <property key="staticMaximumValue" type="integer" defaultValue="100">
+            <caption>Maximum value static</caption>
+            <category>Values</category>
+            <description/>
+        </property>
         <property key="bootstrapStyleAttribute" type="attribute" required="false">
             <caption>Bar style attribute</caption>
-            <category>Data source</category>
+            <category>Values</category>
```

Comment:
Move to category 'Appearance', after 'Bar style'

Reply:
Done